js/src/components/modals/SemRushUpsellAlert.js

```diff
@@ -0,0 +1,42 @@
+/* External dependencies */
+import React from "react";
+import { __, sprintf } from "@wordpress/i18n";
+
+/* Internal dependencies */
+import { Alert } from "@yoast/components";
+import { makeOutboundLink } from "@yoast/helpers";
+
+const PremiumLandingPageLink = makeOutboundLink();
+
+/**
+ * Creates the content for the Yoast SEO Premium upsell alert in SEMrush modal
+ *
+ * @returns {React.Element} The Yoast SEO Premium upsell alert
+ */
+const SemRushUpsellAlert = () => {
+
+	return (
```

Comment:
Running `grunt eslint` locally, it tells me:
`16:34  error  Block must not be padded by blank lines`